libs/ui/src/precinct_scanner_tally_qrcode.tsx

```diff
@@ -58,7 +58,7 @@ export function PrecinctScannerTallyQrCode({
         }
 
         setResultsReportingUrl(
-          `https://results.voting.works/?p=${encodeURIComponent(
+          `${election.quickResultsReportingUrl}/?p=${encodeURIComponent(
```

Comment:
Any concern here about the URL having a trailing slash?

Reply:
I don't think I want us to try to be too smart about correcting this... maybe we issue a warning?

Reply:
or maybe we make it part of `zod` constraints?

Reply:
I like the idea of making it part of the zod validation if that's easy!

Reply:
done!